### PR TITLE
[MBL-16917][Student] Grades screen LGO

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: 
- Verify that the quantitative data is not shown on the grades screen when restrict quantitative data is turned on. Neither on the total grade and neither on the assignment grades.
- Verify that nothing changed on the grades screen when quantitative data is not restricted.

refs: MBL-16917
affects: Student
release note: none

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/interaction/CourseGradesInteractionTest.kt [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+        setUpCustomGrade(score = 100.0, data = data, restrictQuantitativeData = false)
+        goToGrades(data)
+        courseGradesPage.assertTotalGrade(ViewMatchers.withText("100%"))
+    }
+
+    @Test
+    @TestMetaData(Priority.IMPORTANT, FeatureCategory.GRADES, TestCategory.INTERACTION, false)
+    fun testGradeIsDisplayedWithOnlyGradeWhenQuantitativeDataIsRestricted() {
+        val data = setUpData(courseCount = 1, favoriteCourseCount = 1)
+        setUpCustomGrade("A", 100.0, data, true)
```
Comment (kdeakinstructure) [inline apps/student/src/androidTest/java/com/instructure/student/ui/interaction/CourseGradesInteractionTest.kt:66]: could u use the boolean parameter name here just as in many other cases? (restrictQuantitiveData).

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Review by hermannakos — APPROVED** [review]
QA + 1

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**